lib/Stats.tsx

```diff
@@ -104,6 +104,7 @@ interface StatsProps {
   readonly parameters: VapixParameters
   readonly videoProperties: VideoProperties
   readonly refresh: number
+  readonly volume?: number
```

Comment:
Can be removed (in case if it's possible to use `volume` from `videoProperties` instead)